orangecanvas/application/addons.py

```diff
@@ -66,12 +66,12 @@ def normalize_name(name):
 
 
 def prettify_name(name):
-    dash_split = name.split('-')
-    # Orange3-ImageAnalytics => ImageAnalytics
-    if len(dash_split) > 1 and dash_split[0].lower() in ['orange', 'orange3']:
-        name = '-'.join(dash_split[1:])
+    # orange, separator, addon-name
+    orange, _, addon = name.partition('-')
```

Comment:
This turns `"Survival-Analysis"` into `Analysis`. Shouldn't the first part be discarded only when it's Orange or Orange3?